hack/.vendor-helpers.sh

```diff
@@ -70,15 +70,17 @@ clean() {
 		github.com/docker/docker/integration-cli # external tests
 	)
 
-	local dockerPlatforms=( linux/amd64 $(_dockerfile_env DOCKER_CROSSPLATFORMS) )
+	local dockerPlatforms=( linux/amd64 windows/amd64 $(_dockerfile_env DOCKER_CROSSPLATFORMS) )
```

Comment:
Right, but this isn't reading the environment variable, it's actually reading the `Dockerfile` and scraping the environment variable:
```console
$ source hack/.vendor-helpers.sh
$ _dockerfile_env DOCKER_CROSSPLATFORMS
	linux/386 linux/arm 	darwin/amd64 darwin/386 	freebsd/amd64 freebsd/386 freebsd/arm 	windows/amd64 windows/386
```